src/client/components/delta/delta.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../imports';
```

Comment:
Missing header, could we add lint rule for SCSS?

Reply:
I don't see such rule in sass-lint.

Reply:
:( so we need to remember the rule and check during review